.github/workflows/publish.yml

```
@@ -9,38 +9,173 @@ env:
+          node-version: 22
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: ${{ matrix.settings.target }}
+      - name: Build
+        run: npm run build:native --workspaces --if-present -- ${{ matrix.settings.target }}
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: bindings-${{ matrix.settings.target }}
+          path: bandersnatch/npm/${{ matrix.settings.package }}/*.node
```

actionlint (1.7.10)

property "package" is not defined in object type {host: string; target: string}

(expression)